olive/engine/packaging/packaging_generator.py

```diff
@@ -296,11 +296,17 @@ def _package_candidate_models(
         tempdir = Path(temp_dir)
 
         if packaging_type == PackagingType.Zipfile:
+            best_node: FootprintNode = _get_best_candidate_node(pf_footprints, footprints)
+            model_attributes = best_node.model_config["config"].get("model_attributes") or {}
+
             if packaging_config.include_sample_code:
-                _package_sample_code(Path(__file__).parent, tempdir)
+                _package_sample_code(Path(__file__).parent, tempdir, model_attributes.get("is_generative", False))
```

Comment:
Nit: we can abstract `is_generative` to a separate method.

Reply:
Done!